pkg/volume/test_util_linux.go

```diff
@@ -0,0 +1,30 @@
+// +build linux
+
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
If you end up going with `if runtime.GOOS == ...` in the test, then this package won't be needed, and you can get rid of these files.